client/src/components/dashboard/Community/UserCard.js

```diff
@@ -71,8 +91,10 @@ class UserCard extends React.Component {
 
     const {
       user,
+      user: {
```

Comment:
You have user in here twice, though it looks like one is a var expansion and one is a string? If that's the case you should make the string explicit by wrapping it in quotes, otherwise `user` is getting overloaded.